index.bs

```diff
@@ -874,17 +873,29 @@ Given a [=request=] |request|:
 
 <h3 dfn id="does-filter-data-match">Does filter data match</h3>
 
+To <dfn>match two [=attribution filter value|filter values=]</dfn> |a| and |b| given a [=boolean=] |isNegated|:
```

Comment:
I wonder if this section would be easier to understand if we defined two different algorithms for negation?

For example:

To <dfn>match filter values</dfn> given a [=filter value=] |a| and a [=filter value=] |b|:
...

To <dfn>match filter values with negation</dfn> given a [=filter value=] |a| and a [=filter value=] |b|:

Then we could just switch/branch in the match filter data algorithm. wdyt?